From cd67e8f39cf0ff3ee3912da6f31059ffb0f8f65d Mon Sep 17 00:00:00 2001
From: Andres Rodriguez <andres.rodriguez@amd.com>
Date: Wed, 31 Aug 2016 17:39:55 -0400
Subject: [PATCH 5504/5855] drm/amdkfd: make Makefile paths relative

Use relative paths in Makefile for consistency with amdgpu

BUG: KFD-279
Change-Id: If5bfe26498ba941941bec65b10896d6d340af0a7
Signed-off-by: Andres Rodriguez <andres.rodriguez@amd.com>

 Conflicts:
	drivers/gpu/drm/amd/amdkfd/Makefile
---
 drivers/gpu/drm/amd/amdkfd/Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/Makefile b/drivers/gpu/drm/amd/amdkfd/Makefile
index 3d2acf9..61d3e18 100644
--- a/drivers/gpu/drm/amd/amdkfd/Makefile
+++ b/drivers/gpu/drm/amd/amdkfd/Makefile
@@ -2,8 +2,10 @@
 # Makefile for Heterogenous System Architecture support for AMD GPU devices
 #
 
-ccflags-y := -Idrivers/gpu/drm/amd/include/  \
-		-Idrivers/gpu/drm/amd/include/asic_reg
+FULL_AMD_PATH=$(src)/..
+
+ccflags-y := -I$(FULL_AMD_PATH)/include/  \
+		-I$(FULL_AMD_PATH)/include/asic_reg
 
 amdkfd-y	:= kfd_module.o kfd_device.o kfd_chardev.o kfd_topology.o \
 		kfd_pasid.o kfd_doorbell.o kfd_flat_memory.o \
-- 
2.7.4

